pkg/pillar/scripts/agent-start.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+agent=$1
+shift
+
+# to start the agent inside service we touch the file
+while [ ! -f "/run/$agent.start" ]; do sleep 5; done
+
```

Comment:
This seems excessively ugly. We should have explicit dependencies using pubsub instead.

Reply:
I couldn't find the like button, but I whole heartedly agree.